pkg/nuctl/command/create.go

```diff
@@ -87,27 +87,17 @@ func newCreateProjectCommandeer(createCommandeer *createCommandeer) *createProje
 			commandeer.projectConfig.Meta.Namespace = createCommandeer.rootCommandeer.namespace
 
 			return createCommandeer.rootCommandeer.platform.CreateProject(&platform.CreateProjectOptions{
-				ProjectConfig: commandeer.projectConfig,
+				ProjectConfig: &commandeer.projectConfig,
 			})
 		},
 	}
 
-	cmd.Flags().StringVar(&commandeer.projectConfig.Spec.DisplayName, "display-name", "", "Project display name, if different than name")
-	cmd.Flags().MarkDeprecated("display-name", "will be removed on the next major version release") // nolint: errcheck
```

Comment:
Add to description that display name flag no longer accepted

Reply:
added to command description